docs/_js/ErrorCodesComponent.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+/* global React ReactDOM errorMap:true */
+'use strict';
+
+function replaceArgs(msg, argList) {
+  let argIdx = 0;
+  return msg.replace(/%s/g, function() {
+    const arg = argList[argIdx++];
+    return arg === undefined ? '[missing argument]' : arg;
+  });
+}
+
+function segmentify(str) {
+  const urlRegex = /(https:\/\/fb\.me\/[a-z\-]+)/g;
```

Comment:
How is `fb.me` involved here? I think I'm probably just missing a step here. Ultimately I'd like to not use `fb.me` for any of this since we have to register the links and I don't think it supports query params.